lib/chewy.rb

```diff
@@ -19,6 +19,27 @@
 
 require 'elasticsearch'
 
+def try_require(path)
+  require path
+rescue LoadError
+  nil
+end
+
+try_require 'kaminari'
```

Comment:
[`try_require 'kaminari'`](https://github.com/kaminari/kaminari/blob/master/lib/kaminari.rb) will require `kaminari/core` and `kaminari/activerecord` as well. Can you remove this line?